db/package.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I thought for deployment it is needed. Maybe we can check with @mofterdinger?

Reply:
The `cds build` should be able to generate this during the HANA artifacts build. @chgeo might be able to explain this in more detail and if this is meanwhile feasible for Java projects as well.

Reply:
`cds build` does that already in Node.js projects that typically configure the build result to `gen/`, so that these files can be generated w/o polluting the source tree.
In Java that's different with the 'inplace' build.  We are aware of this glitch, but don't have an answer for now.

Reply:
So we should/need to  leave the `db/package.json` in at the moment for the Cloud Platform Deployment in the single tenant use-case?